test/src/components/universalSearch/universalSearch.spec.jsx

```diff
@@ -76,6 +141,22 @@ const updatedStubLocation = {
 };
 
 describe('<UniversalSearch />', () => {
+    let server;
+
+    before(() => {
+        server = new MockAdapter(axios);
+        server.onGet('/api/traces/searchableKeys').reply(200, stubSearchableKeys);
+        server.onGet('/api/alerts/root-service/unhealthyCount').reply(200, 0);
+        server.onGet('/api/services').reply(200, stubServices);
+        server.onGet('/api/operations?serviceName=root-service').reply(200, stubOperations);
+        server.onGet(/^\/api\/traces\?/g).reply(200, stubTraces);
+        server.onGet(/^\/api\/traces\/timeline\?/g).reply(200, []);
+    });
+
+    after(() => {
+        server = null;
+    });
+    
```

Comment:
Universal Search had a lot of unhandled promise rejection errors during test. This is because when the tests mounted the universal search component, the component made calls to API end points that were not set up. I made some stub api end points here, which resolved the errors.